docs/content/preview/develop/learn/acid-transactions-ycql.md

```diff
@@ -35,107 +35,25 @@ A transaction is a sequence of operations performed as a single logical unit of
 
 - **Consistency** A completed transaction leaves the database in a consistent internal state. This can either be all the operations in the transactions succeeding or none of them succeeding.
 
-- **Isolation** This property determines how/when changes made by one transaction become visible to the other. For example, a *serializable* isolation level guarantees that two concurrent transactions appear as if one executed after the other (that is, as if they occur in a completely isolated fashion). YugabyteDB supports *Snapshot*, *Serializable*, and *Read Committed* isolation levels. Read more about the different [levels of isolation](../../../architecture/transactions/isolation-levels/).
+- **Isolation** This property determines how/when changes made by one transaction become visible to the other. For example, a *serializable* isolation level guarantees that two concurrent transactions appear as if one executed after the other (that is, as if they occur in a completely isolated fashion). YugabyteDB supports *Snapshot* isolation level in the YCQL API. Read more about the different [levels of isolation](../../../architecture/transactions/isolation-levels/).
 
 - **Durability** The results of the transaction are permanently stored in the system. The modifications must persist even in the instance of power loss or system failures.
 
-## Creating the table
+{{<note title="Note">}}
+Although YugabyteDB supports only *Snapshot* isolation level in the YCQL API, it supports three levels of isolation in the [YSQL](../../../explore/transactions/isolation-levels) API: *Snapshot*, *Serializable*, and *Read Committed*
+{{</note>}}
+
+## Transactions property
 
 The table should be created with the `transactions` property enabled. The statement should look something as follows.
```

Comment:
why?

Reply:
Every time there is a mention of YCQL isolation in discussions, it is always mentioned that YCQL supports only Snapshot , but YSQL supports three ..

Reply:
No, under Transactions property, I was wondering why "The table should be created with the `transactions` property enabled". There's no context. Are you saying that when creating tables in YCQL, the transactions property must be set to true if you want to have transactions? (why would you not?).

Reply:
Yes. To enable transactions in YCQL , the table should be created with the transactions property. Or else it wont work.

Reply:
OK the transition seemed abrupt, suggestion below

Reply:
Yes. I noticed that too. Let's revisit the page & restructure it later.